scripts/ynh_install_ruby__2

```diff
@@ -0,0 +1,310 @@
+#!/bin/bash
+
+ynh_ruby_try_bash_extension() {
+  if [ -x src/configure ]; then
+	src/configure && make -C src || {
+	  ynh_print_info --message="Optional bash extension failed to build, but things will still work normally."
+	}
+  fi
+}
+
+rbenv_install_dir="/opt/rbenv"
+ruby_version_path="$rbenv_install_dir/versions"
+# RBENV_ROOT is the directory of rbenv, it needs to be loaded as a environment variable.
+export RBENV_ROOT="$rbenv_install_dir"
+export rbenv_root="$rbenv_install_dir"
+
+ruby_dependencies=""
+build_ruby_dependencies="libjemalloc-dev curl build-essential libreadline-dev zlib1g-dev libsqlite3-dev libssl-dev libxml2-dev libxslt-dev autoconf automake bison libtool"
+pkg_dependencies="$pkg_dependencies $ruby_dependencies"
+build_pkg_dependencies="$build_pkg_dependencies $build_ruby_dependencies"
```

Comment:
Eeeh, bit confused by where these are used ? o.O
Or they are just part of what's in the manifest.toml ? Maybe we don't really need to worry that much about this idk

Reply:
It's just from the legacy helpers. From what I can see they are in the manifest too.